cadctl/cmd/investigate/investigate.go

```diff
@@ -40,19 +41,42 @@ var InvestigateCmd = &cobra.Command{
 	Use:          "investigate",
 	SilenceUsage: true,
 	Short:        "Filter for and investigate supported alerts",
-	RunE:         run,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		var err error
+
+		if config.Config.ConfigFile != "" {
+			// loads the configuration unless config file is present
+			// config should be used only when developing
+			if _, err = config.Config.GetCadOcmClientID(cmd); err != nil {
+				return fmt.Errorf("failed to get cadOcmClientID: %w", err)
+			}
+			if _, err = config.Config.GetCadOcmClientSecret(cmd); err != nil {
+				return fmt.Errorf("failed to get cadOcmClientSecret: %w", err)
+			}
+			if _, err = config.Config.GetCadOcmURL(cmd); err != nil {
+				return fmt.Errorf("failed to get cadOcmURL: %w", err)
+			}
+		}
+
+		if _, err = config.Config.GetExperimental(cmd); err != nil {
+			return fmt.Errorf("failed to get experimental flag: %w", err)
+		}
+
+		return nil
```

Comment:
Why not just a `config, err := config.Load()` wrapping all of this? We're also introducing a new antipattern (global `config` variable) here, can we instead use a properly scoped variable here? 

If we want to specifically load only what we need because we think it will be different for manually triggered stuff, we could have two configuration types that could be loaded.